devito/arch/compiler.py

```diff
@@ -443,6 +443,9 @@ def __init_finalize__(self, **kwargs):
             self.cflags = ['-mcpu=native'] + self.cflags
         elif isinstance(platform, Graviton):
             self.cflags = ['-mcpu=%s' % platform.march] + self.cflags
+        elif isinstance(platform, Cortex):
+            self.cflags += ['-march=%s' % platform.march]
```

Comment:
Nitpick: could be fstring

Reply:
So I guess for homogeneity I should change the whole file? If this is a transition devito codebase is doing I am happy to do it.